lib/components/form/call-taker/mode-dropdown.tsx

```diff
@@ -23,21 +24,39 @@ type Props = {
  * rental companies).
  */
 function ModeDropdown({ modeDropdownOptions, onChangeModes }: Props) {
-  const [selectedMode, setSelectedMode] = useState(modeDropdownOptions[0].label)
+  const [selectedMode, setSelectedMode] = useQueryParam(
+    modeDropdownOptions[0].label
+  )
 
   const onChange = React.useCallback(
     (e: React.FocusEvent<HTMLSelectElement>) => {
       setSelectedMode(e.target.value)
-      const newModes = modeDropdownOptions.find(
-        (mdo) => mdo.label === e.target.value
-      )?.combination
-      onChangeModes(newModes || [])
     },
     []
   )
 
+  useEffect(() => {
+    if (selectedMode === undefined) {
+      setSelectedMode(modeDropdownOptions?.[0].label)
+      return
+    }
+
+    const newModes = modeDropdownOptions.find(
+      (mdo) => mdo.label === selectedMode
+    )?.combination
+    onChangeModes(newModes || [])
+  }, [selectedMode, modeDropdownOptions, onChangeModes])
+
   return (
-    <select onBlur={onChange} onChange={onChange} value={selectedMode}>
+    <select
+      onBlur={onChange}
+      onChange={onChange}
+      value={
+        typeof selectedMode === 'string' && selectedMode !== undefined
```

Comment:
same question here

Reply:
if selectedMode is null or 0 we don't want this behavior!

Reply:
Is the `string` type check enough here?

Reply:
Good catch thanks!